package.json

```
@@ -50,6 +50,22 @@
     "vitepress": "^0.11.4",
     "yorkie": "^2.0.0"
   },
+  "peerDependencies": {
+    "sass": "^1.32.7",
+    "stylus": "^0.54.8",
+    "less": "^4.1.1"
```

```suggestion
    "sass": ">=1.0.0",
    "stylus": ">=0.54.0",
    "less": ">=4.0.0"
```

Like this? So we don't force users to upgrade their dependencies.